internal/notifier/azure_eventhub.go

```diff
@@ -17,10 +17,19 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"net/url"
+	"strings"
 
 	"github.com/Azure/azure-amqp-common-go/v4/auth"
 	eventhub "github.com/Azure/azure-event-hubs-go/v3"
+	"github.com/fluxcd/notification-controller/api/v1beta3"
 	eventv1 "github.com/fluxcd/pkg/apis/event/v1beta1"
+	pkgauth "github.com/fluxcd/pkg/auth"
+	"github.com/fluxcd/pkg/auth/azure"
+	"github.com/fluxcd/pkg/cache"
+	pkgcache "github.com/fluxcd/pkg/cache"
+	"k8s.io/apimachinery/pkg/types"
```

Comment:
Let's reorganize the imports in four blocks in the following order:

1. Standard libraries
2. External libraries
3. `fluxcd/pkg` libraries
4. notification-controller libraries